CMSgov/onemac
# Oy2 19510 - update waiver amendment id format

Story: https://qmacbis.atlassian.net/browse/OY2-19510
Endpoint: https://d2nomt2lppm75o.cloudfront.net/

### Details

As a OneMAC State User, I need the approved Waiver ID formats for 1915(b) Waiver Renewals to be explained and validated so that errors in the submission are decreased and OneMAC and SEATool are more aligned.

### Changes

- updated validation and messaging for waiver amendments
- Added FAQ section for waiver amendment and updated link on submission form

NOTE: Originally the story wanted to remove the original FAQ section for waiver formats, however, waiver rai and TE FAQs still point to that section. It was decided to leave the original section in until all usage is replaced.

### Test Plan

1. Login as state submitter
2. From submission Dashboard do a New Submission
3. Choose Waiver Action
4. On Waiver form select Waiver Amendment Type Action
5. Validate ID hint text reflects new format
6. Type an invalid format and verify the error message reflects the new format and submit is disabled.
7. Type a valid format amendment number that already exists in the system and verify the error message and submit is disabled.
8. Type a valid format amendment number that has no base/renewal in the system and verify warning message
9. Type a valid format amendment number that does have a base/renewal in the system and verify no error or warning message appears
10. Verify submit succeeds with valid amendment number when warning message is present
11. Verify submit succeeds with valid amendment number when no error or warning message is present
